modules/nf-core/parabricks/mutectcaller/main.nf

```diff
@@ -1,8 +1,10 @@
 process PARABRICKS_MUTECTCALLER {
     tag "$meta.id"
     label 'process_high'
+    label 'process_gpu'
+    stageInMode 'copy' // needed by the module to work properly - can be removed once fixed upstream
```

Comment:
Do you have a PR that could be linked to? Then it makes it easy to see whether it has been addressed.